modules/ROOT/pages/migration-example-complex.adoc

```diff
@@ -1135,7 +1135,7 @@ To return a specific HTTP Status code, instead of setting a `http.status` proper
 ----
 
 .delete-product-flow in Studio 7
-image::m4-apikit-based-example-delete-product-flow.png[delete-product-flow in Studio 7]
+image::m4-apikit-based-example-delete-product-flow.png["A flow showing the process for deleting a product, including error handling and HTTP status setting"]
```

Comment:
```suggestion
image::m4-apikit-based-example-delete-product-flow.png["A flow showing the process for deleting a product, including error handling and HTTP status setting"] Should **E**rror be initial cap since other instances are?
```